lib/locales/br/frontend/user.json

```diff
@@ -137,5 +137,6 @@
     "description": "Como usuário convidado você terá acesso limitado à plataforma como júri, estando associado apenas ao hackathon <1>{{challengeName}}</1>"
   },
   "organizations": "Organizações",
-  "followers": "Seguidores"
+  "followers": "Seguidores",
```

Comment:
This key is generic and may be used more in the future. What do you think about moving it to the `common`?